myscopefile.txt

```diff
@@ -0,0 +1 @@
+192.168.2.0/24
```

Comment:
This can't be committed. If you need to put files in for testing deployments, put them in a `tmp` directory, which are ignored by .gitignore.

Reply:
I didn't see an issue with having an example file.

Reply:
We can (and frankly we should, I'm not sure why I never thought of that) provide examples in the documentation, but it shouldn't be placed here, and the presence of this will result in anyone who runs your docker compose to automatically have the agent(s) start scanning this subnet range. We, as developers, should never be triggering, or making it easy to accidentally trigger, scans that the consumer did not specify.

You could add a section to the server README for now that addresses what valid lines of a scope file look like, but I don't think this is the right place for it.

Reply:
Removed